packages/iov-cli/src/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { TSError } from "ts-node";
+import { Script } from "vm";
+
+export function executeJavaScript(code: string, filename: string) {
+  const script = new Script(code, { filename: filename });
+  return script.runInThisContext();
```

Comment:
I'm still wondering where this "context" comes from, with all the variables declared in the repl. Seems a bit magic, but maybe this is a module variable from vm?